flycheck.el

```diff
@@ -9877,11 +9877,24 @@ See URL `http://call-cc.org/'."
          "Note: " (zero-or-more not-newline) ":\n"
          (one-or-more (any space)) "(" (file-name) ":" line ") " (message)
          line-end)
+   (warning line-start
+            "Warning: " (zero-or-more not-newline) ",\n"
+            (one-or-more (any space)) (zero-or-more not-newline) ":\n"
+            (one-or-more (any space)) "(" (file-name) ":" line ") " (message)
+            line-end)
    (warning line-start
             "Warning: " (zero-or-more not-newline) ":\n"
             (one-or-more (any space)) "(" (file-name) ":" line ") " (message)
             line-end)
    (error line-start "Error: (line " line ") " (message) line-end)
+   ;; A of version 4.12.0, the chicken compiler doesn't provide a
+   ;; line number for this error.
+   (error line-start "Error: "
```

Comment:
I'd prefer if we moved that pattern just after the other "Error: " one (the last one), since it would be easier to compare the two.

Reply:
Done. Please take another look.